language/en/email/topic_notify.txt

```diff
@@ -1,8 +1,8 @@
-Subject: Topic reply notification - "{TOPIC_TITLE}"
+Subject: Reply in "{TOPIC_TITLE}"
 
 Hello {USERNAME},
 
-You are receiving this notification because you are watching the topic "{TOPIC_TITLE}" at "{SITENAME}". This topic has received a reply<!-- IF AUTHOR_NAME != '' --> by {AUTHOR_NAME}<!-- ENDIF --> since your last visit. No more notifications will be sent until you visit the topic.
+The topic "{TOPIC_TITLE}" received a new reply<!-- IF AUTHOR_NAME != '' --> by {AUTHOR_NAME}<!-- ENDIF --> since your last visit to "{SITENAME}". No more emails will be sent until you visit the topic.
 
 If you want to view the newest post made since your last visit, click the following link:
```

Comment:
Sollte so lauten: 

> If you want to view the newest post click the following link:

Reply:
Also in der Änderungsdatei ist die Zeile unverändert:

> If you want to view the newest post made since your last visit, click the following link:
> {U_NEWEST_POST}

Reply:
Doch: https://gist.github.com/marc1706/fbb369c966ed7f07bf0dabbf6900cff8#file-language_changes_3-3-3_to_3-3-4-rc1-diff-L570

Reply:
Mhh - da schein ich wohl verrutscht zu sein